libcontainer/configs/validate/validator_test.go

```diff
@@ -842,3 +842,32 @@ func TestValidateScheduler(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateIOPriority(t *testing.T) {
+	testCases := []struct {
+		isErr    bool
+		priority int
+	}{
+		{isErr: false, priority: 0},
+		{isErr: false, priority: 7},
+		{isErr: true, priority: -1},
+	}
+
+	for _, tc := range testCases {
+		ioPriroty := configs.IOPriority{
+			Priority: tc.priority,
+		}
+		config := &configs.Config{
+			Rootfs:     "/var",
```

Comment:
(This value looks unusual, but it doesn’t matter here)

Reply:
It seems we need to set an existing dir. I tried to delete this line and got following error.
```
 validator_test.go|869| iopriority: 0, expected nil, got error invalid rootfs: stat : no such file or directory
```